codeship_yaml/main.py

```diff
@@ -77,17 +79,21 @@ def color_message(message, color=None):
     elif color == 'red':
         color = '\033[91m'
     else:
-        return message
+        color = None
```

Comment:
😢  unfortunately this isn't Ruby.

Will update `color` to be an empty string. The escape code shouldn't matter, it just stops any current highlighting, so it's safe to leave in there.